code/modules/antagonists/cult/blood_magic.dm

```diff
@@ -354,17 +354,19 @@
 	var/uses = 1
 	var/health_cost = 0 //The amount of health taken from the user when invoking the spell
 	var/datum/action/innate/cult/blood_spell/source
-
-/obj/item/melee/blood_magic/Initialize(mapload, spell)
+/obj/item/melee/blood_magic/Initialize(loc, var/spell)
```

Comment:
Mapload should always be the first var in Initialize(). I don't see where you're even using `loc` but if you do need it then fix the order of the args of whatever `new()` call.

Reply:
i just forgot to remove loc after the conversion i am putting mapload now in trough